src/components/connection-health/connection-health.css

```diff
@@ -22,4 +22,23 @@
     align-items: center;
     justify-content: center;
     align-items: center;
+}
+
+[data-text-dir="rtl"] {
```

Comment:
I think it's dengerous
need a variables instead

Reply:
Switched to dir attribute, instead of custom one. 
Checked multiple websites most of them use dir , but why is this dangerous?